lib/kafka/consumer.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require "kafka/consumer_group"
+require "kafka/consumer_interceptors"
```

Comment:
This is an out of scope comment but have you considered using `ActiveSupport/Callbacks`? Most people using this lib use Rails and we are a ruby shop.

You can use it without loading all of Rails: Just add `activesupport` as a gem.

Then you could define your interceptors as callbacks and register on this class.

-  https://api.rubyonrails.org/classes/ActiveSupport/Callbacks.html